Annoyingly, the test failure looks (technically) real. Kind of wild that lxml isn't a 100% drop-in replacement.
```
  ======================================================================
  FAIL: corehq.apps.ota.tests.test_search_claim_endpoints:CaseClaimEndpointTests.test_search_endpoint
  ----------------------------------------------------------------------
  Traceback (most recent call last):
    File "/usr/local/lib/python3.9/unittest/mock.py", line 1336, in patched
      return func(*newargs, **newkeywargs)
    File "/mnt/commcare-hq-ro/corehq/apps/ota/tests/test_search_claim_endpoints.py", line 263, in test_search_endpoint
      self._assert_empty_search_result(response, params)
    File "/mnt/commcare-hq-ro/corehq/apps/ota/tests/test_search_claim_endpoints.py", line 296, in _assert_empty_search_result
      self.assertEqual('<results id="case" />', response.content.decode('utf-8'), message)
  AssertionError: '<results id="case" />' != '<results id="case"/>'
  - <results id="case" />
  ?                   -
  + <results id="case"/>
   : {'name': 'Jamie Face', 'case_type': 'case'}
```

--- a/corehq/apps/ota/tests/test_search_claim_endpoints.py
+++ b/corehq/apps/ota/tests/test_search_claim_endpoints.py
@@ -293,7 +293,7 @@ class CaseClaimEndpointTests(TestCase):
 
     def _assert_empty_search_result(self, response, message=None):
         self.assertEqual(response.status_code, 200, message)
-        self.assertEqual('<results id="case" />', response.content.decode('utf-8'), message)
+        self.assertEqual('<results id="case"/>', response.content.decode('utf-8'), message)
 
     def test_duplicate_claim_after_case_changes(self):
         """

